packages/cactus-plugin-ledger-connector-iroha2/package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "@hyperledger/cactus-plugin-ledger-connector-iroha2",
+  "version": "1.0.0",
```

Comment:
@outSH Same thing here about the version number needing a manual bump after the 1.1.0 release, apologies.